templates/layouts/auth.html.ep

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- --- Start SEO (Search Engine Optimization) --- -->
+
+    <title>Control Nest - <%= title %></title>
+    <meta name="description" content="Your Data’s Cozy Nest" />
+
+    <!-- Charset -->
+    <meta charset="utf-8" />
+
+    <!-- Viewport for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <!-- Robot Tag (DO NOT INDEX) -->
+    <meta name="robots" content="noindex, nofollow" />
+
+    <!-- --- End SEO (Search Engine Optimization) --- -->
+
+    <!-- Website Icon -->
+    <!-- <link rel="icon" href="/img/favicon.ico" /> -->
+
+    <!-- --- Start CSS (Cascading Style Sheets) --- -->
+
+    <!-- Font Awesome V6 -->
+    %= stylesheet '/css/fontawesome.min.css'
+
+    <!-- Normalize CSS -->
+    %= stylesheet '/css/normalize.min.css'
+
+    <!-- AdminLTE v3.2 -->
+    %= stylesheet '/css/adminlte.min.css'
+
+    <!-- --- End CSS (Cascading Style Sheets) --- -->
+  </head>
+
+  <!-- Some other pages will use this exact layout, but with a different body class -->
+  <% my $body_class = stash('auth_layout') // 'login-page bg-body-secondary'; %>
```

Comment:
This links with the Auth controller, should find a solution for both cases, login and lock screen.. and for the future, register